libraries/bxdf/standard_surface.mtlx

```diff
@@ -19,33 +19,33 @@
             doc="The directional bias of reflected and transmitted light resulting in materials appearing rougher or glossier in certain directions." />
-      <input name="specular_rotation" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
+      <input name="specular_rotation" type="float" value="0" uisoftmin="0.0" uisoftmax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
             doc="Rotation of the axis of specular anisotropy around the surface normal." />
       <input name="transmission" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission" uifolder="Transmission" uiadvanced="true"
             doc="Transmission of light through the surface for materials such as glass or water. The greater the value the more transparent the material." />
       <input name="transmission_color" type="color3" value="1, 1, 1" uimin="0,0,0" uimax="1,1,1" uiname="Transmission Color" uifolder="Transmission" uiadvanced="true"
             doc="Color tint on the transmitted light." />
-      <input name="transmission_depth" type="float" value="0" uimin="0.0" uimax="100.0" uiname="Transmission Depth" uifolder="Transmission" uiadvanced="true"
+      <input name="transmission_depth" type="float" value="0" uimin="0.0" uisoftmax="100.0" uiname="Transmission Depth" uifolder="Transmission" uiadvanced="true"
             doc="Specifies the distance light travels inside the material before its becomes exactly the transmission color according to Beer's law." />
       <input name="transmission_scatter" type="color3" value="0, 0, 0" uimin="0,0,0" uimax="1,1,1" uiname="Transmission Scatter" uifolder="Transmission" uiadvanced="true"
             doc="Scattering coefficient of the interior medium. Suitable for a large body of liquid or one that is fairly thick, such as an ocean, honey, ice, or frosted glass." />
       <input name="transmission_scatter_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission Anisotropy" uifolder="Transmission" uiadvanced="true"
             doc="The amount of directional bias, or anisotropy, of the scattering." />
-      <input name="transmission_dispersion" type="float" value="0" uimin="0.0" uimax="100.0" uiname="Transmission Dispersion" uifolder="Transmission" uiadvanced="true"
+      <input name="transmission_dispersion" type="float" value="0" uimin="0.0" uisoftmax="100.0" uiname="Transmission Dispersion" uifolder="Transmission" uiadvanced="true"
             doc="Dispersion amount, describing how much the index of refraction varies across wavelengths." />
-      <input name="transmission_extra_roughness" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission Roughness" uifolder="Transmission" uiadvanced="true"
+      <input name="transmission_extra_roughness" type="float" value="0" uisoftmin="0.0" uisoftmax="1.0" uiname="Transmission Roughness" uifolder="Transmission" uiadvanced="true"
```

Comment:
This should be [-1, 1] hard range. It's an add term to specular roughness which itself is [0,1].